conan/cli/args.py

```diff
@@ -124,13 +133,15 @@ def common_graph_args(subparser):
                                 "directory when no --requires or --tool-requires is "
                                 "given",
                            default=None)
-    add_reference_args(subparser)
-    subparser.add_argument("--requires", action="append",
-                           help='Directly provide requires instead of a conanfile')
-    subparser.add_argument("--tool-requires", action='append',
-                           help='Directly provide tool-requires instead of a conanfile')
-    add_common_install_arguments(subparser)
-    add_lockfile_args(subparser)
+    group = subparser.add_argument_group("common graph options")
+    group.add_argument("--requires", action="append",
+                       help='Directly provide requires instead of a conanfile')
+    group.add_argument("--tool-requires", action='append',
+                       help='Directly provide tool-requires instead of a conanfile')
+    ref_group = add_reference_args(subparser)
+    install_group, profile_group = add_common_install_arguments(subparser, group)
+    lockfile_group = add_lockfile_args(subparser)
+    return ref_group, install_group, profile_group, lockfile_group
```

Comment:
As can be seen here, we now return the groups so that users can add to them in their custom commands, but maybe we should be using named tuples? Right now adding a new one/changing the order would break users